src/DotVVM.Framework/Configuration/HtmlAttributeTransformConfiguration.cs

```diff
@@ -27,7 +37,10 @@ public HtmlAttributeTransformConfiguration()
 
         public IHtmlAttributeTransformer GetInstance()
         {
-            return instance.Value;
+            if (isFrozen)
+                return instance.Value;
+            else
+                throw new NotSupportedException("Can not use HtmlAttributeTransformConfiguration since it can still be modified");
```

Comment:
"This HtmlAttributeTransformConfiguration must be frozen before the IHtmlAttributeTransformer instance can be returned." would IMHO be a better message.